assets/css/_search_form.scss

```diff
@@ -1,95 +1,97 @@
-.c-search-form__row {
+.c-search-form {
   display: flex;
+  flex-flow: column nowrap;
+  gap: 1rem;
+}
 
-  &:not(:last-child) {
-    margin-bottom: 1rem;
-  }
+.c-search-form__search-control {
+  // Make control area match designs
+  height: 2.25rem;
 }
 
-.c-search-form__text {
+.c-search-form__search-input-container {
+  // Layout floating `input-controls` over the `search-input-container`
   position: relative;
-  flex: 1 1 auto;
+
+  // Use bootstrap control as base for now
+  @extend .form-control;
+
+  // fill control defined valid area
+  height: 100%;
+  // Make space for `input-controls`
+  padding-right: calc(2.25rem * 2);
+  // Ignore bootstrap padding due to `input-controls`
+  padding-block: 0;
+
+  border-color: $color-gray-400;
+
+  &:focus,
+  &:focus-within,
+  &:active {
+    border: 1px solid $color-eggplant-500;
+    box-shadow: 0px 0px 0px 2px #aa4ef22b;
+    color: $color-gray-800;
+  }
 }
 
 .c-search-form__input {
+  // Align self within parent boundaries
   width: 100%;
-  height: 2.375rem;
-  box-sizing: border-box;
+  height: 100%;
 
-  border-radius: 4px 0 0 4px;
-  border: none;
+  // Match font size from the Route Picker Search
+  @include font-small;
 
-  padding: 7px 0 7px 13px;
+  // Remove default displays
+  border: none;
+}
 
-  font-size: var(--font-size-s);
-  font-weight: var(--font-weight-reg);
-  color: var(--font-color-on-light-600);
+// #region Search Input Buttons
+.c-search-form__input-controls {
+  position: absolute;
+  // Stick to right side of box, do not intercept mouse events to the text input
+  inset-block: 0;
+  right: 0;
+  width: 0;
 
-  &:focus {
-    border: 1px solid $color-eggplant-500;
-  }
+  // Layout controls from the right side
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: end;
 }
 
 .c-search-form__clear {
   @include button-icon(0.875rem);
-  right: 0.75rem;
-  position: absolute;
-  top: 0.75rem;
-
-  color: $color-gray-600;
 
-  &:hover,
-  &:focus-within {
-    color: $color-eggplant-500;
-  }
+  // Reserve space around icon in flex container
+  flex: 0;
+  height: 100%;
+  aspect-ratio: 1/1;
 }
 
 .c-search-form__submit {
   @include button-icon(1rem);
+  color: $color-eggplant-700;
 
-  flex: 1 0 auto;
-  max-width: 2.625rem;
-  max-height: 2.375rem;
-
-  border-radius: 0 4px 4px 0;
-
-  display: flex;
-
-  justify-content: center;
-  align-items: center;
-}
+  &[disabled] {
+    color: var(--ui-grays-gray-700, #3c3f4c);
+  }
 
-.c-search-form__property-buttons {
-  display: flex;
-  flex: 1 0 auto;
-  list-style-type: none;
-  margin: 0;
-}
+  // flex: 0 0 2.25rem;
+  // flex: 0;
```

Comment:
**nitpick:** looks like this might be some temporary code that got left over as comments.